### ISSUE TITLE
 StartMenuHelper64.dll is an unsigned .dll - can this be signed?

### ISSUE THREAD

**Issue by justageekhere** [Open-Shell/Open-Shell-Menu#287]
The current version (4.4.142) of OpenShell is flagged on my system (Win10) due to an included unsigned DLL - StartMenuHelper64.dll.

This may be one of the unsigned DLLs that is causing me at every boot to have to step through several "blue screens" in order to select "Disable driver signature enforcement", at which point booting continues just fine and dandy. 

Windows' built-in SigVerif finds no unsigned drivers. Really?

SysInternals SigCheck flags this OpenShell .DLL as well as 10 others that I'm working my way through - including a Microsoft driver that SigVerif does not find ( ! ). 

Versions:
 - Open-Shell: 4.4.142
 - OS: Windows 10 1909

**Comment by ge0rdi** [comment]
> The current version (4.4.142) of OpenShell is flagged on my system (Win10) due to an included unsigned DLL - StartMenuHelper64.dll.

How exactly is OpenShell `flagged` on your system?

> This may be one of the unsigned DLLs that is causing me at every boot to have to step through several "blue screens" in order to select "Disable driver signature enforcement", at which point booting continues just fine and dandy.

`Open-Shell` doesn't uses any drivers (kernel-mode components), so it is improbable that those BSODs are related to its (unsiged) DLLs.

Did you try to uninstall `Open-Shell` to see if it helps?

> can this be signed?

Currently we are not signing our binaries, because this is voluntary project and code signing certificate isn't free (one needs to pay for it periodically).
So unless we'll get a certificate and make sure we can pay for it, we can't sign binaries :(

**Comment by justageekhere** [comment]
First let me say that OpenShell is a wonderful thing and that Win10 would be considerably less tolerable without it. I was a long-time user of ClassicStart as well. 

I should clarify several things about "flagged on my system" by which I mean the following. 

My system will not finish booting until after I have stepped through the boot-time "blue screens" (for lack of a better description... they are titled things like "Recovery" and "Startup Settings" and are definitely not BSOD) and selected "Disable driver signature enforcement" on the Startup Settings screen. 

Having to do this appears to be due to unsigned .DLLs in Windows\system32, so I went looking for unsigned .DLLs. Microsoft's SigVerif found none so it was no help.

I cast a wider net and found Microsoft SysInternals SigCheck which found 11 unsigned .DLLs in the same directory - one of which was StartMenuHelper64.dll. Thus my posted question about signing it.

I will uninstall OpenShell and see if that makes any difference. I really doubt it, which means I will continue to use OpenShell and be happy. I'll be fiddling my way though the other 10 unsigned .DLLs and hope for success.

Thanks.
